src/OSPSuite.Core/Services/PredictedVsObservedChartService.cs

```diff
@@ -107,6 +114,77 @@ public void ConfigureAxesDimensionAndTitle(IReadOnlyList<DataColumn> observation
          chart.UpdateAxesVisibility();
       }
 
+      public IEnumerable<DataRepository> AddDeviationLine(float foldValue, List<DataColumn> observationColumns, PredictedVsObservedChart chart,
+         int numberOfDeviationLines)
+      {
+         var deviationCurves = addDeviationLines(foldValue, observationColumns, chart, numberOfDeviationLines).ToList();
+         chart.UpdateAxesVisibility();
+         return deviationCurves;
+      }
+
+      private IEnumerable<DataRepository> addDeviationLines(float foldValue, IReadOnlyList<DataColumn> observationColumns,
+         PredictedVsObservedChart chart, int numberOfDeviationLines)
+      {
+         var dataColumns = observationColumns;
+
+         var uniqueDimensions = dataColumns.Select(dataColumn => _dimensionFactory.MergedDimensionFor(dataColumn))
+            .DistinctBy(dimension =>
+               dimension.DisplayName); //We are using display name here as it is the only way to identify unique merge dimensions
+
+         foreach (var mergedDimension in uniqueDimensions)
+         {
+            yield return createFoldDeviationRepositories(foldValue, chart, dataColumns, mergedDimension, numberOfDeviationLines);
+         }
+      }
+
+      private DataRepository createFoldDeviationRepositories(float foldValue, PredictedVsObservedChart chart,
+         IReadOnlyList<DataColumn> dataColumns,
+         IDimension mergedDimension, int numberOfDeviationLines)
+      {
+         var deviationLineRepository= createDeviationRepository(foldValue, dataColumns, mergedDimension);
+         chart.AddCurvesFor(deviationLineRepository, x => x.Name, _dimensionFactory,
+            (column, curve) => curve.UpdateDeviationCurve(column.Name, numberOfDeviationLines));
+
+         return deviationLineRepository;
+      }
+
+      private DataRepository createDeviationRepository(float foldValue, IReadOnlyList<DataColumn> dataColumns,
+         IDimension mergedDimension)
```

Comment:
call this one dimension. DO we care that this is a mergedDimension?
Also why is this parameter on a new line? Is resharper formatting like this?